src/HomeWork_8/HomeWork_8.ts

```diff
@@ -0,0 +1,91 @@
+// Вам необхідно написати додаток Todo list. У списку нотаток повинні
+// бути методи для додавання нового запису, видалення, редагування та
+// отримання повної інформації про нотатку за ідентифікатором, а так само
+//  отримання списку всіх нотаток.Крім цього, у користувача має бути
+// можливість позначити нотаток, як виконаний, і отримання інформації про те,
+//  скільки всього нотаток у списку і скільки залишилося невиконаними.Нотатки не повинні бути порожніми.
+// Кожний нотаток має назву, зміст, дату створення і редагування та статус.
+// Нотатки бувають двох типів.Дефолтні та такі, які вимагають підтвердження при ридагуванні.
+// Окремо необхідно розширити поведінку списку та додати можливість пошуку нотатка за ім'ям або змістом.
+// Також окремо необхідно розширити список можливістю сортування нотаток за статусом або часом створення.
```

Comment:
Please, pay attention to this requirement. That functionality should be implemented separately from the basic Todo list class.